frontend/src/pages/NotFoundPage.module.css

```diff
@@ -0,0 +1,29 @@
+.error {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.errorIcon {
+  height: 10rem;
+  width: 10rem;
+  color: var(--ion-color-secondary);
+}
+
+.errorCode {
+  margin: 0;
+  text-align: center;
+  font-size: 4rem;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
Is there a specific reason to use a specific different font than everywhere else?

Reply:
Font removed.